rust/arrow/src/ipc/reader.rs

```diff
@@ -939,12 +939,13 @@ mod tests {
     use flate2::read::GzDecoder;
 
     use crate::util::integration_util::*;
-    use std::env;
     use std::fs::File;
 
+    use crate::util::test_data_dir::ARROW_TEST_DATA;
+
     #[test]
     fn read_generated_files() {
-        let testdata = env::var("ARROW_TEST_DATA").expect("ARROW_TEST_DATA not defined");
```

Comment:
Could you post example output for when an example of test is run and the test data cannot be found? I expected the unwrap does show the detailed error message but just want to be sure since we're reminving these explicit messages here.

Reply:
@andygrove Make sense, let's me refined the errors.